src/modules/verifiable-credentials/storage-client.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  PresentationDefinition,
+  VerifiableCredential,
+  CredentialSubject,
+} from '@ew-did-registry/credentials-interface';
+import { AxiosInstance } from 'axios';
+import { CacheClient } from '../cache-client';
+
+export default class VCStorageClient {
+  private _http: AxiosInstance;
+
+  constructor(cacheClient: CacheClient) {
+    this._http = cacheClient.http;
+  }
+
+  async getCredentialsByPresentationDefinition(
+    definition: PresentationDefinition
+  ): Promise<VerifiableCredential<CredentialSubject>[]> {
+    const { data } = await this._http.post<
+      VerifiableCredential<CredentialSubject>[]
+    >(`/vp/match`, definition);
+
+    return data;
```

Comment:
I think this section is incorrect. I think that `/vp/match` is returning the a `MatchResultsDto` which has more than just `VerifiableCredentials`. 
https://identitycache-dev.energyweb.org/api/#/Verifiable%20Presentation/PresentationExchangeController_getMatchedCredentialsForVpDefinition
```suggestion
  async getCredentialsByPresentationDefinition(
    definition: PresentationDefinition
  ): Promise<SelectResults> {
    const { data } = await this._http.post<
      SelectResults
    >(`/vp/match`, definition);

    return data;
  }
```